### PR TITLE
Add Extension Manifest Converter

### PR DESCRIPTION
**Pull request by solomonkinard**
Add Extension Manifest Converter.

### CODE REVIEW

File: site/en/blog/new-in-extensions-2/index.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+---
+layout: 'layouts/blog-post.njk'
+title: An example blog post
+description: >
+  Open source tool to convert extensions to Manifest V3. You’ll still need to manually update any service worker code that relies on a DOM, in addition to updating chrome.scripts.
```
Comment (samthor) [inline site/en/blog/new-in-extensions-2/index.md:5]: I don't know exactly where we landed on this—I have strong opinions—but right now we don't allow smart quotes (’). Please replace with ' or ".

Comment (solomonkinard) [reply site/en/blog/new-in-extensions-2/index.md:5]: SG

File: site/en/blog/enabling-shared-array-buffer/index.md [hunk 2]
```diff
@@ -50,8 +56,33 @@ can gather data on requests that failed as a result of
...
+break integrations that require cross-origin window interactions such as OAuth
+and payments. To mitigate this problem, we are [exploring relaxing the
+condition](https://github.com/whatwg/html/issues/6364) to enable cross-origin
+isolation to `Cross-Origin-Opener-Policy: same-origin-allow-popups`. This way
+the communication with the window opened by itself will be possible.
+
+If you want to enable cross-origin isolation to use `SharedArrayBuffer` but are
+blocked by these challenges, we recommend [registering for an origin
+trial](#origin-trial) and waiting until the new modes are available. We are not
+planning to terminate the origin trial until these new modes are available. {%
```
Comment (samthor) [inline site/en/blog/enabling-shared-array-buffer/index.md:84]: can you put endAside on its own line?

File: site/en/blog/enabling-shared-array-buffer/index.md [hunk 3]
```diff
@@ -7,11 +7,11 @@ description: >
   SharedArrayBuffer will arrive in Android Chrome 88. It will only be available
   to pages that are cross-origin isolated. Starting in Desktop Chrome 91 it will
   also only be available to cross-origin isolated pages. You can register for an
-  origin trial to retain the current behavior until Desktop Chrome 93.
+  origin trial to retain the current behavior until Desktop Chrome 96.
```
Comment (samthor) [inline site/en/blog/enabling-shared-array-buffer/index.md:10]: These changes all seem unrelated. I don't mind them going in but is there a reason they're not in another PR?

Comment (dotproto) [reply site/en/blog/enabling-shared-array-buffer/index.md:10]: This may have been a mistake on my part. I wanted to get Solomon's branch up to date, so I merged `main` into his PR branch. That picked up a bunch of unrelated changes. I think I may need to create a new branch off of `main` and cherry pick his commits in order to undo what I did.

File: site/en/blog/enabling-shared-array-buffer/index.md (hunk 2 above)
Comment (dotproto) [reply site/en/blog/enabling-shared-array-buffer/index.md:84]: That should probably be handled in a separate PR as this content has already been merged into main.

**Comment by dotproto** [comment]
Created #699 to address the issues created by my bad merge in @solomonkinard's PR.
